Mathlib/Order/Birkhoff.lean

```diff
@@ -162,6 +176,7 @@ powerset lattice. -/
 noncomputable def birkhoffFinset : α ↪o Finset {a : α // SupIrred a} := by
   exact (birkhoffSet _).trans Fintype.finsetOrderIsoSet.symm.toOrderEmbedding
 
+
```

Comment:
No need to add more empty lines randomly

Reply:
Thanks!